queries/models/sources.yml

```
@@ -424,3 +424,9 @@ sources:
 
             data_particao = date("{{ now.date() }}")
         loaded_at_field: timestamp(datetime(data_particao, horario), 'America/Sao_Paulo')
+
+  - name: source_inmet
+    database: rj-smtr-dev
+
+    tables:
+      - name: meteorologia
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)